framework/assets/yii.js

```diff
@@ -395,76 +501,18 @@ window.yii = (function ($) {
         return str.replace(/[\-\[\]\/\{\}\(\)\*\+\?\.\\\^\$\|]/g, "\\$&");
     }
 
-    function getHostInfo() {
-        return location.protocol + '//' + location.host;
-    }
-
-    function initScriptFilter() {
-        var hostInfo = getHostInfo();
-        var loadedScripts = {};
-
-        var scripts = $('script[src]').map(function () {
-            return this.src.charAt(0) === '/' ? hostInfo + this.src : this.src;
-        }).toArray();
-        for (var i = 0, len = scripts.length; i < len; i++) {
-            loadedScripts[scripts[i]] = true;
-        }
-
-        $.ajaxPrefilter('script', function (options, originalOptions, xhr) {
-            if (options.dataType == 'jsonp') {
-                return;
-            }
-
-            var url = options.url.charAt(0) === '/' ? hostInfo + options.url : options.url;
-
-            if (url in loadedScripts) {
-                var item = loadedScripts[url];
-
-                // If the concurrent XHR request is running and URL is not reloadable
-                if (item !== true && !isReloadable(url)) {
-                    // Abort the current XHR request when previous finished successfully
-                    item.done(function () {
-                        if (xhr && xhr.readyState !== 4) {
-                            xhr.abort();
-                        }
-                    });
-                    // Or abort previous XHR if the current one is loaded faster
-                    xhr.done(function () {
-                        if (item && item.readyState !== 4) {
-                            item.abort();
-                        }
-                    });
-                } else if (!isReloadable(url)) {
-                    xhr.abort();
-                }
-            } else {
-                loadedScripts[url] = xhr.done(function () {
-                    loadedScripts[url] = true;
-                }).fail(function () {
-                    delete loadedScripts[url];
-                });
-            }
-        });
-
-        $(document).ajaxComplete(function (event, xhr, settings) {
-            var styleSheets = [];
-            $('link[rel=stylesheet]').each(function () {
-                if (isReloadable(this.href)) {
-                    return;
-                }
-                if ($.inArray(this.href, styleSheets) == -1) {
-                    styleSheets.push(this.href)
-                } else {
-                    $(this).remove();
-                }
-            })
-        });
+    /**
+     * Returns absolute url based on the given url
+     * @param {string} url Initial url
+     * @returns {string}
+     */
+    function getAbsoluteUrl(url) {
+        return url.charAt(0) === '/' ? pub.getBaseCurrentUrl() + url : url;
     }
 
     return pub;
-})(jQuery);
+})(window.jQuery);
 
-jQuery(function () {
-    yii.initModule(yii);
+window.jQuery(function () {
+    window.yii.initModule(window.yii);
```

Comment:
Variable `yii` was set using `window.yii`, but then accessed with just `yii`. This is also hard to test.